resources/privacy-and-gdpr.md

```diff
@@ -192,15 +231,11 @@ To retrieve all the identifier data stored locally by the SDK as a JSON string,
 
 ## Configuration keys
 
-To update the SDK configuration, programmatically, use the following information to change your privacy configuration values. For more information, [Configuration API reference](../using-mobile-extensions/mobile-core/configuration/configuration-api-reference.md).
+To update the SDK configuration, programmatically, use the following information to change your default consent values. For more information, [ConsentAPI reference](../using-mobile-extensions/adobe-edge-consent/edge-consent-api-reference.md).
 
 | Key | Description |
 | :--- | :--- |
-| `global.privacy` | Setting to control privacy opt status; values may include `optedid`, `optedout`, `optunknown` |
-
-## Video
-
-{% embed url="https://www.youtube.com/watch?v=kgUJNFQp3PI" caption="Using Mobile SDK Privacy APIs" %}
+| `consent.default` | Defines the set of default consent preferences for the SDK in XDM format. [More info.](https://github.com/adobe/xdm/blob/fc0773107f29928e1dc4753f8f055836083ea53f/docs/reference/mixins/profile/profile-consents.schema.md) |
```

Comment:
can we add an example of how to set it up programmatically?

Reply:
I don't think we should. It's not something we really want to encourage. Defaults should be set from launch.

Reply:
This might be helpful especially for first time launch (install) scenarios, if there is no network connection, you may want to have a fallback config